scalableminds/webknossos
# fix dynamic space direction (#1999)

My PR that fixed the rotation of the view in flight mode, broke the dynamic space direction, because the `setRotationAction` was "misused" to affect the dynamic space direction. I made this more explicit by introducing a new `setDirectionAction` that is used instead.

### Steps to test:
- Press `Space` in orthogonal mode and note the direction in which the z-Value of the position changes. Now set a node, then move backwards using `Space + Shift` and set another node. Now press `Space` again - the direction in which the z-Value of the position changes should be reversed.
- Yes this is a somewhat obscure feature, but it has been there since the beginning and it seems to be quite essential to "advanced" tracers. :)

### Issues:
- fixes #1999 

------
